src/ContainerRegistry/ContainerRegistry/Models/PSImportSourceCredentials.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.Commands.ContainerRegistry.Models
 {
     public class PSImportSourceCredentials
     {
-        public PSImportSourceCredentials(string password, string username = default(string))
+        public PSImportSourceCredentials(string username, string password)
```

Comment:
This change makes more sense but it raises compatibility issue. 
Please check if any other method calls this method. If no, this change is safe.

Reply:
Only one method calls this method. It's safe.